### ISSUE TITLE
when a team is locked, spawn menu does not reflects this fact

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1995]

**Comment by bmorel** [comment]
I thuoght this could be done with the `if` ui element, but the information about locked team is apparently never sent to clients, so until we get a way for client to query server for the information, it can't be done.

**Comment by DolceTriade** [comment]
I was thinking that it would be useful to have a concept of "sync data" in the server, similar to config strings, where certain data (key-value pairs) are always kept in sync between a server and all of its clients. The k,v pairs pairs can be arbitrary and can store arbitrary data.

I think this would solve many problems regarding coordination and can even be used to do fancier things like map votes at the end of a game.

**Comment by necessarily-equal** [comment]
I have an impression that this is already what config strings are about, but it's quite impractical to do at the moment.

**Comment by bmorel** [comment]
AFAIK you can't check config strings from within `.rml` files

**Comment by DolceTriade** [comment]
Yes, but config strings need to be strings and can't be C structs without parsing. Also we only have so many.

This new thing could have a Lua API to make it accessible from .rml files.
